react-native/react/actions/tracker.js

```diff
@@ -18,6 +18,30 @@ type TrackerActionCreator = (dispatch: Dispatch, getState: () => {tracker: RootT
 
 // TODO make actions for all the call back stuff.
 
+export function registerTrackerChangeListener (): (dispatch: Dispatch) => void {
+  return dispatch => {
+    const param = {
+      channels: {
+        tracking: true
+      }
+    }
+    engine.listenOnConnect(() => {
+      engine.rpc('notifyCtl.setNotifications', param, {}, (error, response) => {
+        if (error != null) {
+          console.error('error in toggling notifications: ', error)
+        }
+      })
+    })
+
+    engine.listenGeneralIncomingRpc('keybase.1.NotifyTracking.trackingChanged', function (args) {
```

Comment:
> what happens here if we do this multiple times?

Oh, you found a bug!  But the bug's actually that the functions in this file (registerIdentifyUI, registerPinentryListeners, registerTrackerChangeListener) aren't called often enough -- in particular, if the service restarts without Electron restarting, we won't call these functions again, and so we'll end up without having these UIs registered on our running service.  We should be calling these whenever RPC connects/reconnects.

I've filed DESKTOP-170 for this, since it's a more architectural change.